src/main/java/com/gregtechceu/gtceu/api/machine/MetaMachine.java

```diff
@@ -533,7 +533,7 @@ public boolean shouldRenderGrid(Player player, BlockPos pos, BlockState state, I
         }
 
         if (toolTypes.contains(GTToolType.WRENCH)) {
-            if (player.isShiftKeyDown()) {
+            if (!player.isShiftKeyDown()) {
                 if (isFacingValid(side)) {
```

Comment:
From your screenshots I think multiblock controllers are now missing the center rotation icon, this should fix that:
```suggestion
                if (isFacingValid(side) || (allowExtendedFacing() && hasFrontFacing() side == getFrontFacing()) {
```